src/api/libcellml/model.h

```diff
@@ -366,6 +366,89 @@ class LIBCELLML_EXPORT Model: public ComponentEntity
      */
     bool fixVariableInterfaces();
 
+    /**
+     * @brief Add an import source item to this model.
+     *
+     * Add import source by reference to the model. If the import
```

Comment:
Would replace with "Add the @p importSource to the model" and maybe rephrase the rest of that paragprah accordingly.

Reply:
TBD in #618